fedimint-core/src/config.rs

```diff
@@ -745,12 +748,19 @@ pub trait TypedServerModuleConfig: DeserializeOwned + Serialize {
     }
 }
 
+pub type FrostShareAndPop = (
+    BTreeMap<schnorr_fun::fun::Scalar<Public, NonZero>, schnorr_fun::fun::Scalar<Secret, Zero>>,
+    Signature,
+);
+
 /// Things that a `distributed_gen` config can send between peers
 // TODO: Needs to be modularized in case modules want to send new message types for DKG
 #[derive(Serialize, Deserialize, Debug, Clone)]
 pub enum DkgPeerMsg {
     PublicKey(secp256k1::PublicKey),
     DistributedGen(SupportedDkgMessage),
+    Polynomial(Vec<Point>),
+    ShareAndPop(FrostShareAndPop),
```

Comment:
I am a at a loss here, why is the Dkg modified here? I just checked the frost paper and our dkg should support it already, at least if we modify the trait DkgGroup to allow for a custom scalar for scalar multiplication, not just bls12_381::Scalar. The paper uses the same Dkg by Gennaro that we use, which should work over any group without modifications.

Reply:
CC: @m1sterc001guy

Reply:
No reason other than this seemed easier than modifying DKG. Let me know if you have pointers for how to modify DKG so that other modules can utilize this.